README.md

```diff
@@ -61,7 +61,7 @@ Once we have read in these values and passed them into the fragment shader corre
 
 It is first important to choose a microfacet model to describe how light interacts with a surface. In this project, I use the [Cook-Torrance Model](https://renderman.pixar.com/view/cook-torrance-shader) to compute lighting. However, there is a large difference between doing this based on lights within a scene versus an environment map. With discrete lights, we could just evaluate the BRDF with respect to each light and average the results to obtain the overall color, but this is not ideal if you want a scene to have complex lighting that comes from many sources.
 
-### Environment Maps
+###Environment Maps
```

Comment:
The space between `###` and the header name has disappeared on a few headers, possibly as part of a bad merge.  The space is required for GitHub markdown.